### DISCUSSION TITLE
What about Saleor performance ?

### DISCUSSION THREAD

**Discussion by Sarowar-neo** [saleor/saleor#6083] [Q&A]
**Saleor performance on Saleor site.** 

![Screenshot from 2020-08-31 22-01-42](https://user-images.githubusercontent.com/22500875/91741180-2a188480-ebd6-11ea-96da-8fd8641ee2ce.png)

### **Localhost performance why like this ? :(** 

![Screenshot from 2020-08-31 22-13-43](https://user-images.githubusercontent.com/22500875/91741992-5aacee00-ebd7-11ea-9a6e-924e40817fe3.png)

**Comment by karolkielecki** [comment]
@Sarowar-neo Thanks for pointing this out. Your final performance on production depends on many factors such as your deployment etc. That is only the demo site, and results on local development are usually not so great. However, we'll try to improve the demo results as soon as possible because we have other priorities for now: Saleor 2.11 and 3.0. You can find more details here: https://github.com/mirumee/saleor/projects/12

**Comment by amesas** [comment]
Hi,

And what about performance with huge catalogs? We have  a magento2 store with 350000 SKUs and it's growing up.

We already have a nextjs frontend solution that works great, do you think Saleor could be a good alternative (dashboard + api ) to magento2? We don't do anything crazy with the backend/catalog, just a "size" problem.
